tox.ini

```diff
@@ -37,7 +38,7 @@ commands = py.test deluge
 
 ##############
 # Unit tests
-##############
+############## #    DISPLAY = :99.0
```

Comment:
bendikro did it in testing, the tox changes just need to be reverted now that his test fixes are merged in develop.